src/components/menu/menu.module.scss

```diff
@@ -111,37 +111,72 @@
   &.open {
     transform: translateX(0);
   }
+
+  .activeNavLink {
+    border-left: none;
+    & * {
+      color: $color-green-500;
+    }
+  }
+
+  .externalLinksHeader {
+    color: $color-dark-blue-25;
+    margin: 40px 0 24px 0;
+    @include font-body-11;
+
+    // Horizontal line after the External Links Header
+    &::after {
+      content: '';
+      display: block;
+      margin-left: 7px;
+      height: 1px;
+      background-color: $color-dark-blue-300;
+      width: 131px;
+    }
+  }
+
+  .externalLink {
+    display: flex;
+    align-items: center;
+    margin-left: 0;
+    margin-bottom: 16px;
+    height: 24px;
+    gap: 16px;
+    text-decoration: none;
+    @include font-heading-9;
+
```

Comment:
We need a min-width on the svgs

```suggestion
    svg {
      min-width: 24px;
    }
```

<img width="249" alt="image" src="https://github.com/user-attachments/assets/08f71ddc-5588-46cb-90ff-da40fd0d6349">